cpp/src/cider/common/contrib/cityhash102/src/config.h

```diff
@@ -0,0 +1,125 @@
+/* config.h.  Generated from config.h.in by configure.  */

```

Comment:
why need this file? And seems we didn't apply license check or other check on this file/folder?

Reply:
It's from the hash function lib and excluded from license check.